lib/puma/client.rb

```diff
@@ -218,7 +229,12 @@ def decode_chunk(chunk)
         end
       end
 
-      return false
+      if @in_last_chunk
+        set_ready
+        return true
```

Comment:
unneccessary `return`

Reply:
I think so too but I follow the current style:

  * https://github.com/puma/puma/blob/675344e8609509b0d767ae7680436b3b382d8394/lib/puma/client.rb#L156
  * https://github.com/puma/puma/blob/675344e8609509b0d767ae7680436b3b382d8394/lib/puma/client.rb#L300

There are no `return` cases too:

  * https://github.com/puma/puma/blob/675344e8609509b0d767ae7680436b3b382d8394/lib/puma/client.rb#L336
  * https://github.com/puma/puma/blob/675344e8609509b0d767ae7680436b3b382d8394/lib/puma/client.rb#L372

Which style is Puma style?

Reply:
From here on out (I make the rules now!) no more unnecessary returns.

Reply:
OK. I've removed unnecessary returns from this pull request.